src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -67,4 +70,12 @@ class CourtController(
   fun linkDanglingAppointments() {
     appointmentLinker.linkAppointments()
   }
+
+  @DeleteMapping(consumes = [MediaType.APPLICATION_JSON_VALUE], path = ["/video-link-bookings/{videoBookingId}"])
+  @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Being a DELETE operations there isn't any body to consume. There's no need for a 'consumes' attribute.

Reply:
consumes = [MediaType.APPLICATION_JSON_VALUE]  removed